Sources/XCHammer/XcodeTarget.swift

```diff
@@ -345,8 +345,7 @@ public class XcodeTarget: Hashable, Equatable {
 
     func isAllowableXcodeGenSource(path: Path) -> Bool {
         // These files are handled in other ways.
-        // FIXME: remove filtering plist ( these could be used in source file )
-        return path.extension != "plist" && path.extension != "mobileprovision"
```

Comment:
I can't remember exactly why, but there was some reason `plist` is here. We should see the output of `make run_force`, and additionally update it per your commit.

Reply:
Per our discussion, we verified that `make run` and `make test` succeed after fixing an issue with plist number formatting